tests/v2/validation/snapshot/README.md

```diff
@@ -1,36 +1,64 @@
 # Snapshot
 
+For the snapshot tests, the test has only been tested on RKE2. That does not mean RKE1 or K3S are unsupported, they just have not been tested. These tests are designed to accept an existing cluster that the user has access to. If you do not have a downstream cluster in Rancher, you should create one first before running these tests.
```

Comment:
This test should be able to run on k3s, and should test on rke1 as well.

Reply:
I forgot to update this for K3S. Initially, this PR was meant for RKE2/K3S, with RKE1 going to be handled in a separate PR.